1-run-automation-test.md

```diff
@@ -0,0 +1,123 @@
+# Running Automation test with Kobiton
+This guide will demonstrate how to run the automation test on Kobiton devices with TeamCity through GitHub repository.
+## Table of contents
++ [Prerequisites](#prerequisites)
++ [1. Config TeamCity and integrate with GitHub](#1-config-teamcity-and-integrate-with-github)
++ [2. Config automation test script](#2-config-automation-test-script)
++ [3. Run automation test on Kobiton devices](#3-run-automation-test-on-kobiton-devices)
++ [4. Fetch test session data through Kobiton REST API](#4-fetch-test-session-data-through-kobiton-rest-api)
+## Prerequisites 
+- Java(JRE). Supported are:
+  + Oracle java 8 or higher
+  + OpenJDK 8
+- Kobiton account
+## 1. Config TeamCity and integrate with GitHub
+This part will guide you how to setup and configure TeamCity with GitHub. Skip this step if you have already had TeamCity setup.
+ + For instruction on how to install and configure TeamCity server, follow [this guide](https://confluence.jetbrains.com/display/TCD18/Installing+and+Configuring+the+TeamCity+Server).
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ Integrate GitHub with TeamCity, you can follow [this guide](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services).
```

Comment:
can be replaced with 'Instruction on how to integrate GitHub with TeamCity can be found here (link the `here` word with the link above